demo.ipynb

```diff
@@ -158,6 +158,34 @@
    ]
```

Comment:
Further up in the notebook you call the `interpolate()` method, and store its return value, which is `None`, in a variable called `interpolator` - I think you just want to do `P.interpolate()`, or even skip this and just call `P.approximate()` (which calls interpolate anyway) - right?